submissions/kt-std/friends-app/js/script.js

```diff
@@ -0,0 +1,292 @@
+let FRIENDS_ARRAY = [],
+	INITIAL_FRIENDS_ARRAY = [];
+const USERS_AMOUNT = 24,
+	API_URL = `https://randomuser.me/api/?results=${USERS_AMOUNT}`,
+	CARDS_CONTAINER = document.querySelector(".cards__container"),
+	TOTAL_COUNTER = document.querySelector(".amount");
+
+function getFriends() {
+	fetch(API_URL)
+		.then((response) => {
+			if (response.ok) {
+				return response.json();
+			} else {
+				appendErrorMessage(
+					getResponseErrorMessage(
+						response.status,
+						response.statusText
+					)
+				);
+			}
+		})
+		.then((responseBody) => {
+			if (responseBody !== undefined) {
+				INITIAL_FRIENDS_ARRAY = INITIAL_FRIENDS_ARRAY.concat(
+					flattenFriendProperties(responseBody.results)
+				);
+				FRIENDS_ARRAY = INITIAL_FRIENDS_ARRAY;
+				appendFriendsCards(FRIENDS_ARRAY);
+				setTotalCounter(FRIENDS_ARRAY);
+				initializeAgeLimits(FRIENDS_ARRAY);
+			}
+		})
+		.catch((error) =>
+			appendErrorMessage(`${error} <br> Try to reload the page!`)
+		);
+}
+
+getFriends();
+
+
+function setTotalCounter(friendsArray) {
+	TOTAL_COUNTER.innerText = `${friendsArray.length} Totals`;
+	if (!friendsArray.length) {
+		appendNoResultsMessage();
+	}
+}
+
+function initializeAgeLimits(friendsArray) {
```

Comment:
1. Calls of `getCertainAgeLimit` repeated twice
2. `getCertainAgeLimit` not clear naming. What do you mean under `certain`? Please, replace it with a less opaque name https://learn.javascript.ru/ninja-code#budte-abstraktny-pri-vybore-imeni
3. You are using unnamed strings, which is bad. They repeated - you can misprint them. From the outside it is not understandable WDYM by passing `"min"` as an argument. Please, give names to such strings everywhere in your app.